zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -100,7 +100,7 @@ pub(super) const TESTNET_ACTIVATION_HEIGHTS: &[(block::Height, NetworkUpgrade)]
     (block::Height(584_000), Blossom),
     (block::Height(903_800), Heartwood),
     (block::Height(1_028_500), Canopy),
-    (block::Height(1_599_200), Nu5),
+    (block::Height(1_821_300), Nu5),
```

Comment:
I can't find a source to double-check this, and the ZIP and `zcashd` code haven`t been changed yet:
https://zips.z.cash/zip-0252#nu5-deployment

Reply:
Here is the ZIP PR:
https://github.com/zcash/zips/pull/600/files#diff-b424a91f01c7799201470397654fcf582c045c318b0f91e1e15af4b7fdfc8f78